src/pages/waiting-room-to-car/cat-a-mod1/waiting-room-to-car.cat-a-mod1.page.ts

```diff
@@ -145,6 +150,16 @@ export class WaitingRoomToCarCatAMod1Page extends BasePageComponent {
         select(getTestCategory),
       ),
     };
+
+    const {
+      testCategory$,
+    } = this.pageState;
+
+    this.merged$ = merge(
```

Comment:
don't need to `merge` if its just one subscription

Reply:
We do that in a lot of places. Such as Pass Completion, keeps everything looking and working the same ;)

Reply:
Probably should be kept for consistency